### DISCUSSION TITLE
Some ideas

### DISCUSSION THREAD

**Discussion by Geccub** [idena-network/idena-desktop#668] [Ideas]
Some ideas from the Discord chat:

1. To reduce the problem of "semi-uniqueness" (without shortening the short sessions) more flips could be offered during the short session and human status could be given a 'uniqueness factor' based on the number of flips solved.  Ie if only 6 flips are solved in 2 minutes this could be a uniqueness factor of .5, but solving 8/10/12 flips etc could be .66, .83 and 1 respectively.  All mining and other rewards during the epoch would be reduced by the "uniqueness factor" (with the balance to be burned).  Someone running two "0.5-uniqueness" nodes would receive the same mining and validation rewards as running a single "1-uniqueness" node.  This would reduce the incentive for running multiple nodes and also increase scarcity of iDNA (through the burning of rewards of identities with uniqueness factors less than 1).  This approach should not deter newcomers, as they could still reach "Human" status with "0.5 uniqueness" solving the same number of flips as presently required.  It would, however,  improve the uniqueness and security of the network (Idena's main selling point) and also improve the value of iDNA.

2. Show rank based on age.  Eg, oldest identity would be 1 of 7974.  This would offer an incentive for people to improve their "rank" validation to validation.  Oracles could also a filter for age (ie, only select  identities over a certain rank or in top 10 percent by age).  Ranks could also be given titles (ie Rank 1 could be Idena President, top 10% of ranks could be Idena Captains, etc).

**Comment by aidenaio** [comment]
moving to discussions
